### PR TITLE
Update announcer.py to ask people to report bugs

### PR DESCRIPTION
**Pull request by simotek**
Rather then replying to the generated email we would like people to create new issues on bugzilla.opensuse.org

### CODE REVIEW

File: factory-package-news/announcer.py [hunk 1]
```diff
@@ -54,9 +54,9 @@
 Please check the known defects of this snapshot before upgrading:
 https://openqa.opensuse.org/tests/overview?distri=opensuse&groupid=1&version=Tumbleweed&build={version}
 
-When you reply to report some issues, make sure to change the subject.
-It is not helpful to keep the release announcement subject in a thread
-while discussing a specific problem.
+Please do not reply to this email to report issues, rather file a bug
+on bugzilla.opensuse.org for more information on filing bugs please
```
Comment (DimStar77) [inline factory-package-news/announcer.py]: Feels like there is a full stop missing after 'bugzilla.opensuse.org'

Comment (simotek) [reply factory-package-news/announcer.py]: I thought about putting a comma, but I tend to not like punctuation after url's especially full stops, they tend to break things in places

Comment (DimStar77) [reply factory-package-news/announcer.py]: actually, a full stop would be no problem, as the 'technically correct' name would actually be ```bugzilla.com.``` (the last dot indicating 'DNS root')

Comment (simotek) [reply factory-package-news/announcer.py]: Done, didn't put one at the end of the paragraph though because that does screw up the wiki.

**Comment by jberry-suse** [comment]
RIP review.tumbleweed.boombatower.com

**Comment by jberry-suse** [comment]
Still not sure about this as there is no clear way to designate show-stopping bugs that everyone would like to be aware of via bugzilla. Just get mixed in with the tasks.

**Comment by simotek** [comment]
Yeah I'm still hoping / expecting that people will report really bad issues to the mailing list once there is a bug report, I tried to imply that this is still ok. If the new approach doesn't work the board has discussed creating another list for reporting tumbleweed / leap beta breakages but others wanted to see how well this approach worked first.

**Pull request merged**